src/memfs.ts

```diff
@@ -76,8 +76,9 @@ export class InMemoryFileSystem extends EventEmitter implements ts.ParseConfigHo
 			this.files.set(uri, content);
 		}
 		// Add to directory tree
+		// TODO: convert this to use URIs.
```

Comment:
we can do this later, I already started an implementation of a URI tree a while ago: https://github.com/sourcegraph/javascript-typescript-langserver/blob/uri-tree/src/trees.ts